From c48ef546b80658881579db6389a7489558a0ff0d Mon Sep 17 00:00:00 2001
From: Andreas Cord-Landwehr <cordlandwehr@kde.org>
Date: Mon, 18 Nov 2024 19:50:24 +0100
Subject: [PATCH] Fix absolute path generation into (not installed) header

This fixes a Yocto QA issue that checks that all built artifacts do not
contain any absolute path reference to the build folder. Although this
is not a public header, the absolute patch can cause problems when
splitting the debug symbols and analyzing coredumps on a different
host system.

Upstream-Status: Submitted [https://invent.kde.org/plasma/plasma-workspace/-/merge_requests/4926]
---
 kcms/region_language/localegenhelper/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kcms/region_language/localegenhelper/CMakeLists.txt b/kcms/region_language/localegenhelper/CMakeLists.txt
index 48d01d0b0a..21389d1721 100644
--- a/kcms/region_language/localegenhelper/CMakeLists.txt
+++ b/kcms/region_language/localegenhelper/CMakeLists.txt
@@ -13,7 +13,7 @@ qt_generate_dbus_interface(
     OPTIONS -s -m
 )
 qt_add_dbus_adaptor(localegen_Helper_SRCS ${CMAKE_CURRENT_BINARY_DIR}/org.kde.localegenhelper.xml
-                    ${CMAKE_CURRENT_SOURCE_DIR}/localegenhelper.h LocaleGenHelper)
+                    localegenhelper.h LocaleGenHelper)
 add_executable(plasma-localegen-helper ${localegen_Helper_SRCS})
 target_link_libraries(plasma-localegen-helper
     Qt::DBus
-- 
2.45.2

